haiku-apps/wordclock/wordclock-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An imprecise clock replicant"
+DESCRIPTION="WordClock is imprecise clock replicant and this show clock with word"
+HOMEPAGE="https://github.com/Janus2/WordClock"
+COPYRIGHT="2017 Janus"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="cbb5da9a498ec99a71a68f039042601207eafdd1"
+SOURCE_URI="https://github.com/Janus2/WordClock/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="fe29a3f17467927f88052d9c260b894d8424cf175b633aa2ad22deea2b25abba"
+SOURCE_FILENAME="WordClock-$srcGitRev.tar.gz"
+SOURCE_DIR="WordClock-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	WordClock = $portVersion
+	app:WordClock = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
```

Comment:
WordClock is localized. Currently it was only translated to German. Nevertheless, after the "make $jobArgs", add a line "make bindcatalogs". This will add the localizations to the resources of the executable.